non-msrv-tests/tests/ui/zeroize/zeroize.rs

```diff
@@ -24,28 +24,8 @@ struct WrongCrateSyntax<T>(PhantomData<T>);
 #[derive_where(Zeroize(crate = "struct Test"))]
 struct InvalidCrate<T>(PhantomData<T>);
 
-#[derive(DeriveWhere)]
-#[derive_where(Zeroize(drop, drop))]
-struct DuplicateDrop<T>(PhantomData<T>);
-
```

Comment:
maybe an error when you still have a drop telling you to use ZeroizeOnDrop instead

Reply:
Deprecation notice, good idea, on it.

Reply:
Done.